clang/test/Driver/emit-flang-attrs.f90

```diff
@@ -0,0 +1,10 @@
+// REQUIRES: aarch64-registered-target
```

Comment:
This test should be moved into `clang/test/Driver/flang/`. Also, the other existing flang driver tests should be updated to use `%flang` instead of `%clang --driver-mode=flang`. On the other hand, this seems to conflict with LLVM Flang; have you considered reducing potential conflicts with upstream? We were hoping to support building both LLVM Flang and Classic Flang from the same code base.

Reply:
The moving of this file to flang directory is done by other commit as this commit is authored by someone else and is contributed as-is.

Reply:
OK thanks.